blueprints/reducer/files/__root__/reducers/__name__.js

```diff
@@ -0,0 +1,38 @@
+// __name__ REDUCER
```

Comment:
Just reread your comment and realised maybe I didn't answer it the first time. The file `__name__.js` is the **ember-cli** way of abstracting the _name_ of a type of thing (aka, a "reducer" in this case). Combine that with the `index.js` at the root of the reducer generator and you get an abstraction for directory path between an "addon" and an "app".